package/Dockerfile

```diff
@@ -13,7 +13,9 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL
     chmod +x kubectl
 RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
     curl -sfL https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${ARCH}.tar.gz | tar xvzf -
-RUN curl -sLf ${KUSTOMIZE_URL} | tar -xzf - && chmod +x kustomize
+RUN if [ "${ARCH}" = "amd64" ] || [ "${ARCH}" = "arm64" ]; then \
+    curl -sLf ${KUSTOMIZE_URL} | tar -xzf - && chmod +x kustomize; \
+  fi
```

Comment:
Need to wrap the copy below in a conditional too, as this binary won't exist now. Would any of the other code need to ?

Reply:
Added to multiline copy instead.

Reply:
Looks like it didn't matter as previous build passed too.

Reply:
it didn't pass, it only try doing arm once it has been merged.

Reply:
Also looks like if any fails it fails. Misread k9s part, the binary is curled no matter what the if is just for changing ARCH. Looks like the run will need to be gated.